composer.json

```diff
@@ -16,8 +16,9 @@
         "google/common-protos": "^1.0||^2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.36",
-        "squizlabs/php_codesniffer": "3.*"
+        "phpunit/phpunit": "^4.8.36||^5.5.7||^6.5.14||^7.5.20||^8.5.26",
```

Comment:
More info here, because the gaps only exist with PHPUnit 4 (for PHP 5.5), PHPUnit 5 (for PHP 5.6, 7.0, and 7.1) and PHPUnit 8 (PHP 7.2+), we only need to support _those_ versions of `phpunit`